### PR TITLE
Snowflake

### PR DESCRIPTION
**Pull request by EmilyNecciai**
# Pull Request/Issue Resolution

**Description of Change:**
Adding Snowflake docs.

### CODE REVIEW

File: _docs/_partners/technology_partners/data_and_infrastructure_agility/data_warehouses/snowflake.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+{% endtab %}
+{% endtabs %}
+
+## Braze Benchmarks
+
+Braze Benchmarks will allow Braze prospects and  customers alike to see how they compare to top players in their industry by comparing their metrics against Braze's industry benchmarks.
+
+We created a corresponding Looker Block so you can use Snowflake or Looker to join that data to your data to perform what-if analyses and other, more flexible analytics.
+
+The initial set of top-level categories are: Financial, Health, Weather, Media, Transportation, Business, Cyber, Sports, Government, Lookup Tables, Demographics, Advertising, and Local.
```
Comment (terenceknorr) [inline _docs/_partners/technology_partners/data_and_infrastructure_agility/data_warehouses/snowflake.md]:
```suggestion
The initial industries are: Delivery Services, Ecommerce, Education, Entertainment, Finance, Gaming, Health, Lifestyle, Restaurants, Retail, Technology, Transportation, and Travel.
```

File: _docs/_partners/technology_partners/data_and_infrastructure_agility/data_warehouses/snowflake.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+LIMIT 500
+```
+{% endtab %}
+{% endtabs %}
+
+## Braze Benchmarks
+
+Braze Benchmarks will allow Braze prospects and  customers alike to see how they compare to top players in their industry by comparing their metrics against Braze's industry benchmarks.
+
+We created a corresponding Looker Block so you can use Snowflake or Looker to join that data to your data to perform what-if analyses and other, more flexible analytics.
```
Comment (terenceknorr) [inline _docs/_partners/technology_partners/data_and_infrastructure_agility/data_warehouses/snowflake.md]: The Looker Block line can be removed for now, and replaced with a link to our Braze Benchmarks data:

We created a Data Tool for Braze Benchmarks available here (https://www.braze.com/perspectives/benchmarks), so you can view a selection of the data, outside the Snowflake interface.

File: _docs/_partners/technology_partners/data_and_infrastructure_agility/data_warehouses/snowflake.md [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+
+Braze leverages Snowflake’s Data Exchange to build a presence, find new customers, and expand reach through the ever-growing Snowflake customer base.
+
+{% comment %}
+Learn more about this partnership [here]()!
+{% endcomment %}
+
+### Data Sharing?
+
+Data Sharing allows Braze to allow you secure access to data on our Snowflake portal by sharing view without worrying about workflow friction or slowdown, failure points, and unnecessary costs that come with typical data provider relationships.
```
Comment (terenceknorr) [inline _docs/_partners/technology_partners/data_and_infrastructure_agility/data_warehouses/snowflake.md]:
```suggestion
Data Sharing allows Braze to allow you secure access to data on our Snowflake portal without worrying about workflow friction or slowdown, failure points, and unnecessary costs that come with typical data provider relationships.
```

File: _docs/_partners/technology_partners/data_and_infrastructure_agility/data_warehouses/snowflake.md [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+If you're interested in this integration, reach out to your Braze Account or Customer Success Manager and ask them to consult the Braze Business Intelligence Team on Secure Data Sharing with Snowflake. This will get the cogs going inside Braze and we'll have your views set up in no time!
+
+{% alert important %}
+You must have an account with Snowflake to use the Data Sharing and Braze Benchmark Services.
+{% endalert %}
+
+## Usage & Visualization
+
+Similar to Currents, you can use your Secure Snowflake Data Sharing to...
+- Create complex rile reports,
```
Comment (terenceknorr) [inline _docs/_partners/technology_partners/data_and_infrastructure_agility/data_warehouses/snowflake.md]:
```suggestion
- Create complex reports,
```

File: _docs/_partners/technology_partners/data_and_infrastructure_agility/data_warehouses/snowflake.md [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+### Data Sharing?
+
+Data Sharing allows Braze to allow you secure access to data on our Snowflake portal by sharing view without worrying about workflow friction or slowdown, failure points, and unnecessary costs that come with typical data provider relationships.
+
+## Integration
+
+If you're interested in this integration, reach out to your Braze Account or Customer Success Manager and ask them to consult the Braze Business Intelligence Team on Secure Data Sharing with Snowflake. This will get the cogs going inside Braze and we'll have your views set up in no time!
+
+{% alert important %}
+You must have an account with Snowflake to use the Data Sharing and Braze Benchmark Services.
```
Comment (terenceknorr) [inline _docs/_partners/technology_partners/data_and_infrastructure_agility/data_warehouses/snowflake.md]:
```suggestion
You must have an account with Snowflake to use the Data Sharing Services.
```

File: _docs/_partners/technology_partners/data_and_infrastructure_agility/data_warehouses/snowflake.md [hunk 6]
```diff
@@ -0,0 +1,135 @@
...
+alias: /partners/snowflake/
+---
+
+# Snowflake Secure Data Sharing
+
+Snowflake is a purpose-built SQL cloud data warehouse for all of your data and all of your users. With Snowflake's unique and patented architecture it's easy to amass all of your data, enable rapid analytics, and derive data-driven insights for all of your users.
+
+Braze leverages Snowflake’s Data Exchange to build a presence, find new customers, and expand reach through the ever-growing Snowflake customer base.
+
+Learn more about this partnership [here](braze.com/perspectives/article/snowflake-partner-announcement)!
```
Comment (zzhaobraze) [inline _docs/_partners/technology_partners/data_and_infrastructure_agility/data_warehouses/snowflake.md]: missing the https://

File: _docs/_partners/technology_partners/data_and_infrastructure_agility/data_warehouses/snowflake.md [hunk 7]
```diff
@@ -0,0 +1,135 @@
...
+LIMIT 500
+```
+{% endtab %}
+{% endtabs %}
+
+## Braze Benchmarks
+
+Braze Benchmarks will allow Braze prospects and  customers alike to see how they compare to top players in their industry by comparing their metrics against Braze's industry benchmarks.
+
+We created a [Data Tool for Braze Benchmarks](https://www.braze.com/perspectives/benchmarks) so you can view a selection of the data, outside the Snowflake interface.
```
Comment (zzhaobraze) [inline _docs/_partners/technology_partners/data_and_infrastructure_agility/data_warehouses/snowflake.md]: Not live yet but should be fine if we merge on tueday

**Comment by EmilyNecciai** [comment]
@zzhaobraze Please review and I'll merge.

**Pull request merged**